network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/CachedNetworkStoreClient.java

```diff
@@ -110,6 +110,11 @@ public class CachedNetworkStoreClient extends AbstractForwardingNetworkStoreClie
         delegate::getVoltageLevelDanglingLines,
         delegate::getDanglingLines));
 
+    private final NetworkCollectionIndex<CollectionCache<GroundAttributes>> groundsCache = new NetworkCollectionIndex<>(() -> new CollectionCache<>(
+            delegate::getGround,
+            delegate::getVoltageLevelGrounds,
+            delegate::getGrounds));
+
```

Comment:
I know it's because the other PR got merged before but can you avoid moving the code, so the review is easier and we really see what changed

Reply:
I completely agree... 

It is not about enjoying moving code, but about having code that is more readable and organized. 

I encourage you next time to add the complementary code in the right place, not just to add the code for the grounds at the end of each function. Instead, we can do better by adding the code following the other injection equipment type, 
and that way, we can easily understand that it is an injection.